src/main/java/org/kairosdb/core/datastore/KairosDatastore.java

```diff
@@ -460,9 +456,13 @@ public int getSampleSize() {
                                 tempFile, m_metric.getCacheTime(), m_dataPointFactory);
                         if (cachedResults != null) {
                             returnedRows = cachedResults.getRows();
-                            if (cacheFilesMetricsProvider != null) {
-                                cachedResults.cacheCreatedAt().ifPresent(cacheFilesMetricsProvider::measureSpan);
-                            }
+                            cachedResults.cacheCreatedAt().ifPresent(fileCreateAt -> {
+                                logger.warn(
+                                        "Cache file was created {} seconds ago",
```

Comment:
This may result in a huge amount of logs (up to the number of queries). Why do we need this log line?

Reply:
Sure, it's for debug purposes